src/py42/clients/cases.py

```diff
@@ -0,0 +1,112 @@
+class CasesClient(object):
+    """A client to expose cases API.
+
+    `Rest documenation <https://default-cases.core-int.cloud.code42.com/swagger-ui.html#/Cases>`__
+    """
+
+    def __init__(self, cases_service, cases_file_event_service):
+        self._cases_service = cases_service
+        self._file_events = cases_file_event_service
+
+    @property
+    def file_events(self):
+        """A collection of method for managing cases file events.
+
+        Returns :
+            :class:`py42.services.casesfileevents.FileEventsService`
+        """
+        return self._file_events
+
+    def create(self, name, subject, assignee, description, findings):
```

Comment:
only `name` should be a required argument here.

Reply:
Waiting for access to test API behaviour.

Reply:
Changed